firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -153,30 +153,23 @@ void threadBLEScanner(void *param) {
   BLE.setScanTimeout(5);
 
   while(true){
-
-    int count = BLE.scan(scanResults, SCAN_RESULT_MAX);
-
+    //filter has to be remade each time, as globalDoorId (from eeprom) is usually read while this thread is already in while loop
```

Comment:
Fair enough. Is creating this filter quick? If it's slow, maybe you'd want to do this outside the function and just update your filter when the EEPROM is loaded and when the door ID is updated in the console function. What do you think? Is it worth the extra complication?